api/v1alpha1/commons.go

```diff
@@ -159,6 +159,13 @@ func cleansePath(path string) string {
 	return strings.Trim(strings.ReplaceAll(path, "//", "/"), "/")
 }
 
+func toString(name interface{}) string {
```

Comment:
I need to understand what this is about. face value it does not look good because an interface cannot be safely converted to a string and there is no error management.

Reply:
I need it here api/v1alpha1/pkisecretengineconfig_types.go on the GetExportedPayload.
The type of vault secret data is `map[string]interface{}`
For specific field is "guaranteed" that is a string type.
We could check the typeof before applying the cast. 
Do you have a better way of handling it?